modules/gitops-argo-rollouts-architecture-overview.adoc

```diff
@@ -0,0 +1,69 @@
+
+* Route percentage-based traffic for canary deployments 
+* Forward incoming user traffic by Using `Service` and `Ingress` resources to the correct application version. Depending on the deployment strategy and the traffic management tool used by your application, you can configure Argo Rollouts to split incoming traffic in application instances with specific percentages.
+* Utilize of multiple mechanisms to collect analysis metrics in order to validate the deployment of a new version of an application.
+
+.Argo Rollouts resources
+[options="header"]
+|===
+|Name | Description
+|`Rollout` |The `Rollout` CR is managed by the Argo Rollouts controller. This CR enables the deployment of applications by using canary or blue-green deployment strategies. It replaces the in-built Kubernetes `Deployment` resource.
+|`AnalysisRun` |The `AnalysisRun` CR is managed by the AnalysisRun controller. It is used to perform an analysis and aggregate the results of analysis to guide the user toward the successful deployment delivery of an application. The `AnalysisRun` CR is an instance of  the `AnalysisTemplate` CR.
+
+Argo Rollouts supports two custom Kubernetes resources to perform Analysis - `AnalysisTemplate` and `AnalysisRun`. You can define an `AnalysisTemplate` on a specific Rollout or globally on a cluster to be shared by multiple rollouts as a Cluster`AnalysisTemplate`. The `AnalysisRun` resource is scoped on a specific rollout.
+|`AnalysisTemplate` |The `AnalysisTemplate` CR is a template file that provides instructions on how to query metrics. The result of these instructions is attached to a rollout in the form of the `AnalysisRun` CR. The `AnalysisTemplate` CR can be defined globally on the cluster or on a specific rollout. You can link a list of `AnalysisTemplate` to be used on replica sets by creating an `Experiment` custom resource.
+|`Experiment` | The `Experiment` CR is managed by the Experiment controller. It is used to run short-lived analysis on an application during its deployment to ensure the application is deployed correctly. The `Experiment` CR can be used independently or run as part of Rollouts.
+|`Service` and `Ingress` | Argo Rollouts natively support routing traffic by services and ingresses by using the Service and Ingress controllers.
+|`OpenShift Route resources` | Rollouts perform traffic splitting across different application versions using OpenShift Route resources.
+|`Route` and `VirtualService` |The OpenShift `Route` and {SMProductName} `VirtualService` resources are used to perform traffic splitting across different application versions.
+|`Prometheus metrics` | `AnalysisTemplate` is configured to connect to Prometheus instances, to evaluate the success or failure of one or more metrics.
+|`Kubernetes Job resource metrics` | Rollouts support the use of simple Kubernetes jobs to run analysis on resource metrics. You can verify a successful deployment of an application based on the successful execution of Kubernetes jobs.
```

Comment:
```suggestion
```
Remove this part as explained in this comment: https://github.com/openshift/openshift-docs/pull/76798#discussion_r1633278339

Reply:
Fixed. Added these rows as bullet points in the introduction. Thanks.